site/content/docs/5.0/getting-started/rtl.md

```diff
@@ -174,7 +174,7 @@ Need both LTR and RTL on the same page? Thanks to [RTLCSS String Maps](https://r
       "scope": "*",
       "ignoreCase": false
     }
-  ] }
+  } ]
```

Comment:
Could be suggested to Bootstrap directly ;)

Reply:
it's already fixed in bootstrap : https://github.com/twbs/bootstrap/commit/75fe3244db7cd99a08cd524fa379dcde8fafbced just a merge error I think

Reply:
Yes that was totally my bad...

Reply:
Yes, it was a merge error